runtimes/node-18.0/src/server.js

```diff
@@ -5,20 +5,40 @@ const { text: parseText, json: parseJson, send } = require("micro");
 
 const USER_CODE_PATH = '/usr/local/server/src/function';
 
+const randomId = () => {
+    return `${new Date().getTime().toString(16)}${Math.round(Math.random() * 1000000000).toString(16)}`;
+}
+
+const writeLogs = (id, type, log) => {
+    fs.appendFileSync(`/mnt/logs/${id}_${type}.log`, log);
+}
+
 const server = micro(async (req, res) => {
     try {
         await action(req, res);
     } catch(e) {
-        const logs = [];
-        const errors = [e.stack || e];
-        res.setHeader('x-open-runtimes-logs', encodeURIComponent(logs.join('\n')));
-        res.setHeader('x-open-runtimes-errors', encodeURIComponent(errors.join('\n')));
+        const logging = req.headers[`x-open-runtimes-logging`] ?? 'enabled';
+        const logId = req.headers[`x-open-runtimes-log-id`] ?? randomId();
```

Comment:
Why are we passing an additional logID? we should use the execution ID itself

Reply:
We dont have execution ID in open runtimes yet. Log ID will be execution ID. Do you want to also call it execution ID? I though tit's best to call id "log-id" because it goes with "logging" header. I thought it would be hard to explain why execution-id is used when logging is enabled